frontend/src/metabase/admin/people/selectors.js

```diff
@@ -29,6 +29,12 @@ export const getUsersWithMemberships = createSelector(
     })),
 );
 
+export const getUserMemberships = createSelector(
+  [(_, props) => props.userId, getMemberships],
+  (userId, memberships) =>
+    memberships && _.values(memberships).filter(m => m.user_id === userId),
+);
+
```

Comment:
From our [style guide wiki](https://github.com/metabase/metabase/wiki/Frontend%3A-Style-Guide#additional-javascript-conventions), I see we prefer using underscore functions only for things that are not implemented in the standard library.

> Prefer native Array methods over underscore's. We polyfill all ES6 features. Use Underscore for things that aren't implemented natively.

Should we maybe use `Object.values(memberships)` here?